nanoFramework.Logging.Syslog/SyslogClient.cs

```diff
@@ -0,0 +1,126 @@
+                throw new ArgumentNullException(nameof(localHostname));
+
+            Facility = facility;
+            LocalHostname = localHostname;
+
+            _socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp);
+            _socket.Bind(new IPEndPoint(localAddress ?? IPAddress.Any, localPort));
+            _socket.Connect(endpoint);
+        }
+
+        /// <summary>
+        /// Create a SyslogClient instance that will send RFC3164 compliant messages to a syslof server
+        /// </summary>
+        /// <param name="hostname">DNS name of the syslog server</param>
+        /// <param name="port">Port of the syslog server</param>
+        /// <param name="localHostname">'Hostname' part of the RFC3164 message (if null, use 'nanoframework')</param>
+        /// <param name="facility">Facility used by this logger</param>
+        /// <param name="localAddress">Local IP address to bind socket (if null IPAdress.Any will be used)</param>
+        /// <param name="localPort">Local port to bind socket (0 to choose available port)</param>
+        public SyslogClient(string hostname, int port, string localHostname, Facility facility = default, IPAddress localAddress = null, int localPort = 0)
+            : this(ResolveEndPoint(hostname, port), localHostname, facility, localAddress, localPort)
+        {
+        }
+
+        /// <summary>
+        /// Facility used by this syslog client
+        /// </summary>
+        public Facility Facility { get; }
+        
+        /// <summary>
+        /// Hostname used in the 'Hostname' field in the syslog message
+        /// </summary>
+        public string LocalHostname { get; }
+
+        /// <summary>
+        /// Send a message to the SysLog server
+        /// </summary>
+        /// <param name="severity">Severity in the message</param>
+        /// <param name="messageTag">Message tag (will be followed by ': ' in the message)</param>
+        /// <param name="messageContent">Message content</param>
+        /// <exception cref="ObjectDisposedException"></exception>
+        public void SendMessage(Severity severity, string messageTag, string messageContent) => SendMessage(Facility, severity, LocalHostname, messageTag, messageContent);
+
+        /// <summary>
+        /// Send a generic syslog message to the SysLog server
+        /// </summary>
+        /// <param name="facility">Facility in the message</param>
+        /// <param name="severity">Severity in the message</param>
+        /// <param name="hostname">Hostname in the message</param>
+        /// <param name="messageTag">Message tag (will be followed by ': ' in the message)</param>
+        /// <param name="messageContent">Message content</param>
+        /// <exception cref="ObjectDisposedException"></exception>
+        public void SendMessage(Facility facility,Severity severity, string hostname, string messageTag, string messageContent)
+        {
+            if (_disposed)
```

Comment:
Please use the brackets { and }
Also please leave an extra enter avec the } So this code will then be:
```csharp
if (_disposed)
{
    throw new ObjectDisposedException(nameof(SyslogClient));
}

// We don't send message with Severity.None which isn't a RFC6134 valid value
if (severity != Severity.None)
{
     _socket.Send(FormatUdpMessage(facility, severity, hostname, messageTag, messageContent));
}
```